agents/web-api/src/main/java/edu/jhuapl/dorset/agents/StockAgent.java

```diff
@@ -221,11 +220,19 @@ protected JsonObject processData(String json, String keyWordCompanyName) {
                 List<JsonElement> returnDataJsonList = responseDataArrayList
                         .subList(responseDataArrayList.size() - DAYS_IN_A_MONTH,
                                 responseDataArrayList.size());
-                returnObj.addProperty("data", returnDataJsonList.toString());
-
+              
                 List<JsonElement> returnLabelsJsonList = responseLabelsArrayList
                         .subList(responseLabelsArrayList.size() - DAYS_IN_A_MONTH,
                                 responseLabelsArrayList.size());
+                
+                //format json data
+                ArrayList<String> returnDataJsonListStr = new ArrayList<String>();
+                for (int i = 0 ; i < responseDataArrayList.size(); i++) {
+                    returnDataJsonListStr.add("\"" + responseDataArrayList.get(i) + "\"");
+                } 
+                String jsonData = "{\"" + keyWordCompanyName + "\":" + returnDataJsonList.toString() + "}";
+                
```

Comment:
This looks like it is manually creating json formatted data rather than using Gson. Why?